pkg/pdatatest/pmetrictest/options.go

```diff
@@ -249,6 +249,20 @@ func maskMetricsResourceAttributeValue(metrics pmetric.Metrics, attributeName st
 	}
 }
 
+func ChangeResourceAttributeValue(attributeName string, changeFn func(string) string) CompareMetricsOption {
```

Comment:
This should be renamed to `ChangeMetricsResourceAttributeValue` because we might want to add the same for logs and traces

Reply:
this is part of the pmetrictest package however.